src/Nethermind/Nethermind.BeaconNode/MockedStart/QuickStartClock.cs

```diff
@@ -31,6 +32,8 @@ public QuickStartClock(long clockOffset)
             _adjustment = TimeSpan.FromSeconds(clockOffset);
         }
 
-        public DateTimeOffset UtcNow() => DateTimeOffset.Now + _adjustment;
+        public DateTimeOffset Now() => DateTimeOffset.Now + _adjustment;
```

Comment:
why not a property?

Reply:
I guess to me the semantics of getting the current time is more of a function than a property.

e.g. to avoid people writing code like the following :-

```
if (_clock.UtcNow > store.GenesisTime)
{
  await _forkChoice.OnTick(store, _clock.UtcNow);
}
...
DateTimeOffset nextTime = _clock.UtcNow.AddSeconds(1);

```

Particularly when setting up a test clock that returns values, I find it easier to count each time the function() is called grabs one test value, whereas I just think of property values as being mostly immutable. 

(Example is the testing of worker with a fake clock; there are two calls to UtcNow(), one at the start and one near the end, so the fake test clock needs two values per loop iteration).

Note: The actual worked uses the unix time seconds, so probably wouldn't use the property anyway.

Mostly just a style thing.

Reply:
Hm... still the .net creators disagree with you how they handle Now as a property ;)

Reply:
@LukaszRozmej  but the .net creators are also responsible for DateTime (an absolutely terrible class for times, which should almost never be used)